apps/SageAccountWeb/src/style/_AccountCreatedPage.scss

```diff
@@ -1,11 +1,14 @@
 .AccountCreatedPage {
+  .right-panel {
+    margin-top: 16px;
+  }
```

Comment:
Adjust right panel text alignment as in the [Figma](https://www.figma.com/file/TOcmJ2QHr51v8Huv3wIec9/Sage-Registration-App?node-id=847-22301&t=IiOfZXxjtpJcY2mv-4)